CONTRIBUTING.md

```diff
@@ -115,31 +126,52 @@ If you are working on windows, check out [`jekyll` docs](https://jekyllrb.com/do
 Take a look at an existing [demo file](https://github.com/fabricjs/fabricjs.com/blob/gh-pages/posts/demos/_posts/2020-2-15-custom-control-render.md).
 Create a new file in the same directory (`posts/demos/_posts`) and you're good to go.
 
-## 🚀 Pull Requests
+## ❤️ Helping Out
 
-Fabric is an open source project 🦄 and as such depends on the community.
+Answering questions and addressing issues is a great way to start contributing to fabric.
 
-Genuine effort has made this repository what it is 🌺 Join Us as we strive to make Fabric better.
+- [Issues](../../issues)
+- [Discussions](../../discussions)
+- [Google group](https://groups.google.com/g/fabricjs)
```

Comment:
Didn't we say we were dropping the Google group?

Reply:
I had no idea. Never been there

Reply:
yes we are never there, and we should make it clear.
We don't have bandwidth for 2 discussion places

Reply:
How?
I will remove the link here